### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by malsbat**

### CODE REVIEW

File: src/cbor.c [hunk 1]
```diff
@@ -26,7 +26,7 @@
 #include <assert.h>
 #include <dps/private/cbor.h>
 
-#if CBOR_MAX_STRING_LEN >= RSIZE_MAX_STR
+#if CBOR_MAX_STRING_LEN > RSIZE_MAX_STR
 #error CBOR_MAX_STRING_LEN must be less than RSIZE_MAX_STR (see safe_str_lib.h)
```
Comment (GregBurns) [inline src/cbor.c]: With your change this is now "less than or equal" is that correct - terminated NUL not included in the size?

File: src/multicast/network.c [hunk 2]
```diff
@@ -77,25 +77,37 @@ static void AllocBuffer(uv_handle_t* handle, size_t suggestedSize, uv_buf_t* buf
...
 
-static void OnMcastRx(uv_udp_t* handle, ssize_t nread, const uv_buf_t* buf, const struct sockaddr* addr, unsigned flags)
+static void OnMcastRx(uv_udp_t* handle, ssize_t nread, const uv_buf_t* buf, const struct sockaddr* addr,
+                      unsigned flags)
 {
     DPS_MulticastReceiver* receiver = (DPS_MulticastReceiver*)handle->data;
     DPS_NetEndpoint ep;
 
-    DPS_DBGPRINT("OnMcastRx\n");
+    if (nread == 0 && !addr) {
```
Comment (GregBurns) [inline src/multicast/network.c]: It seems odd that this ever happens, is this just some weird libuv behavior? Maybe add a comment explaining why this isn't a socket error.

Comment (malsbat) [reply src/multicast/network.c]: yes this is libuv specific.  From http://docs.libuv.org/en/v1.x/udp.html:

The receive callback will be called with nread == 0 and addr == NULL when there is nothing to read, and with nread == 0 and addr != NULL when an empty UDP packet is received.

File: src/cbor.c (hunk 1 above)
Comment (malsbat) [reply src/cbor.c]: I'll double check this one

**Pull request merged**